web-frontend/src/hooks/useUserDetailsBoxesAndBoxItems.ts

```diff
@@ -25,16 +24,24 @@ export default function useUserDetailsBoxesAndBoxItems(){
         getUserDetails(token)
             .then(details => setUserDetails(details))
             .catch(() => toast.error("Connection failed to get user details. Please retry."))
-        getSubscriptions(token)
-            .then(subs => setSubscriptions(subs))
-            .catch(() => toast.error("Connection failed to get abonnements. Please retry."))
         getAllPossibleSubscriptions()
```

Comment:
würde "getAllSubscriptions" reichen? 
"possible Subscriptions" klingt, als würde es auch "impossible Subscriptions" geben 
oder meinst du so etwas wie "getAllSubscriptionsIncludingMultiple" oder "...Repeating"?

Reply:
Es gibt Getter, die alleine die Subscriptions des Users holen.

Reply:
In Zeile 35 holt sich z.B. readSubscriptions nur die vom User.